indico/modules/events/editing/client/js/components/EditingManagement.jsx

```diff
@@ -0,0 +1,39 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import manageTagsURL from 'indico-url:event_editing.manage_tags';
+import dashboardURL from 'indico-url:event_editing.dashboard';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import {ManagementPageBackButton, ManagementPageSubTitle} from 'indico/react/components';
+import {Translate} from 'indico/react/i18n';
+
+import EditingManagementDashboard from './EditingManagementDashboard';
+import TagManager from './TagManager';
+
+export default function EditingManagement({eventId}) {
+  return (
+    <Router>
```

Comment:
Wouldn't it be nice if we also included each editable type page in the routes and not reload for those either?

Reply:
yeah i'm looking into that as well; should be easily doable

Reply:
"should be easily doable".. famous last words!

anyway, it's done now, and the PR suddenly became much bigger 🙈